### ISSUE TITLE
Alex Ultimate Script issue

### ISSUE THREAD

**Issue by xKaigetsu** [quisquous/cactbot#769]
Upon updated to the latest cactbot, I have this two errors popping up; and unable to see the timers for the following two raids (o5n, and alexander ultimate):

```
17/11/2019 9:56:50 PM: Info: Cactbot: BrowserConsole: Error parsing JSON from 04-sb/raid/o5n.js: SyntaxError: Invalid regular expression: /^.{14} 14:(?<id>28A3):(?<source>.*?) starts using (?<ability>.*?) on (?<target>.*?)\./: Invalid group (Source: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/OverlayPlugin/cactbot/ui/raidboss/popup-text.js, Line: 42)
17/11/2019 9:56:50 PM: Info: Cactbot: BrowserConsole: Error parsing JSON from 05-shb/ultimate/the_epic_of_alexander.js: SyntaxError: Invalid regular expression: /^.{14} 1[56]:[0-9A-F]{8}:(?<source>Living Liquid):(?<id>4978):(?<ability>.*?):[0-9A-F]{8}:(?<target>.*?):(?<flags>.*?):.*:(?<x>-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?<y>-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?<z>-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?<heading>-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):.*?:?$/: Invalid group (Source: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/OverlayPlugin/cactbot/ui/raidboss/popup-text.js, Line: 42)
```

**Comment by ChristianIvicevic** [comment]
All the groups starting with `(?` are invalid and most likely should be `(?:`. I think it should look as follows `/^.{14} 1[56]:[0-9A-F]{8}:(?:Living Liquid):(?:4978):(?:.?):[0-9A-F]{8}:(?:.?):(?:.?):.:(?:-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?:-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?:-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):(?:-?[0-9]+(?:[.,][0-9]+)?(?:E-?[0-9]+)?):.?:?$/`.

**Comment by ngld** [comment]
The RegExes are fine. They look wrong because the user above didn't escape the error message and GitHub interpreted the `:` characters as start/end markers for italic text.

As far as I can tell, the problem only happens if you're still using the hibiysleep version of OverlayPlugin since the Chromium version it's based on doesn't support the `(?:` syntax.

**Comment by quisquous** [comment]
Well, good thing that the hibiyasleep version is going on.  The answer here is to switch to the ngld overlay plugin.

**Comment by ngld** [comment]
I double checked to be sure and OverlayPlugin 0.3.4.0 seems to be able to parse `(?:` just fine. However, it doesn't support `(?<groupName>`.
I earlier incorrectly assumed that GitHub interpreted the `:` characters but what's happening is actually that the `*` are interpreted as start/end markers for italic text and the group names (`<groupName>`) are stripped out because GitHub doesn't allow arbitrary HTML.
The end result remains the same: `regexes.js` generates Regexes that OverlayPlugin 0.3.4.0 can't parse but newer versions can.

TL;DR: Please format your bug descriptions properly or the devs have to guess what the actual issue is.
@quisquous You can edit posts made by users and fix stuff like this.

**Comment by quisquous** [comment]
Should be "fixed" now that ngld overlayplugin is the default.
